### ISSUE TITLE
[Background Media]: QA: Tooltip of the play button is not getting dismissed with esc key.

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#8996]
**Engineering info:**  
- Innovation Team engineer: @andy-blum 
- Innovation Team JIRA Ticket: https://jsw.ibm.com/browse/HC-2528
- DPO consulting engineer: N/A

-----

### Description

Page Element:
Tooltip of Play button

Issue Description:
Tooltip of the play button is not getting dismissed with esc key.

Expected:
Tooltip of the play button should get dismissed with esc key.

User Impact:
When the additional content in the page is not getting dismissable by the esc key, user may find discomfort in accessing the web content.

### Component(s) impacted

Background Media

### Browser

Firefox

### Carbon for IBM.com version

v1.35.0-rc.0

### Severity

Severity 3 = The problem is visible or noticeable to users but does not impede the usability or functionality. Affects minor functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-react

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components-react

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by andy-blum** [comment]
@proeung 

I'm not sure if this is actually a bug or not, and if it is, it probably needs fixed upstream.

The [carbon-core tooltip documentation _does_ specify that tooltips should be dismissed with `esc` is pressed](https://carbondesignsystem.com/components/tooltip/usage#interactions), and that is the behavior in the [live demo](https://carbondesignsystem.com/components/tooltip/usage#live-demo). However, the [tooltip in carbon-web-components](https://web-components.carbondesignsystem.com/iframe.html?id=components-tooltip--definition&args=&viewMode=story) _does not_ incorporate that behavior

**Comment by andy-blum** [comment]
For additional info/context: that tooltip is simply appearing/disappearing based on `:hover` and `:focus` selectors in the stylesheets, with no js involved at all, so this would likely require reworking in other locations as well such as the [masthead with logo & tooltip](https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components/?path=/story/components-masthead--with-alternate-logo-and-tooltip)

**Comment by proeung** [comment]
Related to https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/9158

**Comment by proeung** [comment]
Blocked by https://github.com/carbon-design-system/carbon-web-components/issues/980
